contents/docs/self-host/deploy/gcp.mdx

```diff
@@ -58,3 +58,62 @@ As a troubleshooting tool, you can allow HTTP access by setting `ingress.gcp.for
 
 ## Uninstalling the chart
 <UninstallingSnippet />
+
+## Clickhouse Configuration
+
+By default clickhouse is provisioned with standard gcp persistent disk. If you want to specify your own persistent volume claim or switch to a different type of disk
+you can specify the volume claim within `values.yaml`.
+
+### To manually provision a disk
+
+Using the gcloud cli tool for provisioning a disk:
+
+```
+gcloud compute disks create pvc-clickhouse --type=pd-ssd --size=2048GB --zone=us-central1-c
+```
+
+### Create the claim
+
+In order to provide the disk to the clickhouse deployment you must first create a persistent volume and claim within the posthog namespace.
```

Comment:
I'm assuming one needs to run `kubectl apply` on this (so create a separate file & run `kubectl apply` on it?), do we want to be explicit here?